charts/px-monitor/templates/grafana/grafana-config.yaml

```diff
@@ -0,0 +1,47 @@
+{{/* Setting defaults if they are omitted. */}}
+{{- $pxMonitorEnabled := .Values.pxmonitor.enabled | default false }}
+{{- $sslEnabled := .Values.pxmonitor.sslEnabled | default false -}}
+{{- if eq $pxMonitorEnabled true }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pxcentral-grafana-ini-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    grafana: portworx
+    app.kubernetes.io/component: pxcentral-grafana
+{{- include "px-monitor.labels" . | nindent 4 }}
+data:
+  grafana.ini: |
```

Comment:
check https://helm.sh/docs/chart_template_guide/accessing_files/

Reply:
This is not a static file and needs to updated based on inputs. so can't add and load from files directory